presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -517,9 +517,11 @@ public static HiveColumnStatistics fromMetastoreApiColumnStatistics(ColumnStatis
         }
```

Comment:
Can you please improve the commit message. In general please fallow https://chris.beams.io/posts/git-commit/.

I would write something like:
```
Skip Hive boolean column count statistics when of them is set to -1

Impala generates boolean count statistics with -1 for number of false values and
1 for number of true values disregarding actual statistics.
```

Reply:
OK. I have modified the commit message and `push -f`

Reply:
`Skip Hive boolean column count statistics when of them is set to -1` -> `Skip Hive boolean column count statistics when one of them is set to -1`

There was missing `one` word in my comment.